examples/web-wagmi/src/profiles/UseProfileOwnedBy.tsx

```diff
@@ -0,0 +1,49 @@
+import { EthereumAddress, useProfilesOwnedBy } from '@lens-protocol/react';
+
+import { LoginButton } from '../components/auth/LoginButton';
+import { WhenLoggedInWithProfile, WhenLoggedOut } from '../components/auth/auth';
+import { Loading } from '../components/loading/Loading';
+import { useInfiniteScroll } from '../hooks/useInfiniteScroll';
+import { ProfileCard } from './components/ProfileCard';
+
+type UseProfilesOwnedByProps = {
+  address: EthereumAddress;
+};
+
+function UseProfilesOwnedByActiveWallet({ address }: UseProfilesOwnedByProps) {
```

Comment:
nit: naming wise we don't know in this component if `address` is an active wallet

Reply:
I just assumed cause it's private to this module and only active wallet address is fed into it.